### DISCUSSION TITLE
After delete Accordion from component

### DISCUSSION THREAD

**Discussion by uigywnkiub** [heroui-inc/heroui#3428] [Help]
<img width="1112" alt="image" src="https://github.com/nextui-org/nextui/assets/29861553/3fffbbe6-ae89-43fe-ad85-ffcfd313a087">

**Comment by wingkwong** [comment]
Please provide a minimal reproducible environment.

**Reply by uigywnkiub** [reply]
> Please provide a minimal reproducible environment.

I just refactored the code and the bug appeared, after reloading vscode and running with yarn dev without --turbo it is gone. After some time I ran yarn dev --turbo and all was ok.
